yb-voyager/src/srcdb/postgres.go

```diff
@@ -271,9 +271,15 @@ func (pg *PostgreSQL) GetConnectionUriWithoutPassword() string {
 	return sourceUrl.String()
 }
 
-func (pg *PostgreSQL) ExportSchema(exportDir string) {
-	pg.checkSchemasExists()
-	pgdumpExtractSchema(pg.source, pg.GetConnectionUriWithoutPassword(), exportDir)
+func (pg *PostgreSQL) ExportSchema(exportDir string, schemaDir string) {
+	if utils.FileOrFolderExists(filepath.Join(schemaDir, "schema.sql")) {
+		// case for assess-migration cmd workflow
+		log.Infof("directly parsing the '%s/schema.sql' file", schemaDir)
+		parseSchemaFile(exportDir, schemaDir, pg.source.ExportObjectTypeList)
+	} else {
+		pg.checkSchemasExists()
+		pgdumpExtractSchema(pg.source, pg.GetConnectionUriWithoutPassword(), exportDir, schemaDir)
```

Comment:
`pgdumpExtractSchema()` is doing two tasks: dumping the schema and then parsing it. Move the call to `parseSchemaFile()` out of the `pgdumpExtractSchema()`.

Reply:
done